boa/src/syntax/parser/tests.rs

```diff
@@ -12,7 +12,6 @@ use crate::syntax::ast::{
 
 /// Checks that the given JavaScript string gives the expected expression.
 #[allow(clippy::unwrap_used)]
-#[track_caller]
```

Comment:
This is ok for now, but we want this as soon as Rust 1.46 is released (27th August). It helps a lot when debugging tests.

Reply:
Is it coming in 1.46? That is good